honeybee_radiance_folder/folderutil.py

```diff
@@ -118,104 +125,103 @@ def from_dict(cls, input_dict):
+            'Failed to find default file for %s' % self.identifier
         assert os.path.isfile(os.path.join(folder, self.direct)), \
-            'Failed to find direct file for %s' % self.name
+            'Failed to find direct file for %s' % self.identifier
         if self.black is not None:
             assert os.path.isfile(os.path.join(folder, self.black)), \
-                'Failed to find black file for %s' % self.name
+                'Failed to find black file for %s' % self.identifier
         if self.tmtx is not None:
-            # find the root folder
-            try:
-                root, _ = folder.split('model')
-            except ValueError:
-                raise ValueError(
-                    'Invalid input folder. See Radiance folder structure.'
-                    '\nSee https://github.com/ladybug-tools/radiance-folder-structure#radiance-folder-structure'
-                )
-            assert os.path.isfile(os.path.join(root, 'model', 'bsdf', self.tmtx)), \
-                'Failed to find tmtx file for %s' % self.name
+            assert os.path.isfile(os.path.join(bsdf_folder, self.tmtx)), \
+                'Failed to find tmtx file for %s' % self.identifier
         if self.vmtx is not None:
             assert os.path.isfile(os.path.join(folder, self.vmtx)), \
-                'Failed to find vmtx file for %s' % self.name
+                'Failed to find vmtx file for %s' % self.identifier
         if self.dmtx is not None:
             assert os.path.isfile(os.path.join(folder, self.dmtx)), \
-                'Failed to find dmtx file for %s' % self.name
+                'Failed to find dmtx file for %s' % self.identifier
 
     def __repr__(self):
-        return 'ApertureState: {}'.format(self.name)
+        return 'ApertureState: {}'.format(self.identifier)
 
 
-class DynamicNonAperture(object):
-    """Representation of a Dynamic nonaperture geometry in Radiance folder.
+class DynamicScene(object):
+    """Representation of a Dynamic scene geometry in Radiance folder.
 
     Args:
-        name (str): Text string for a unique dynamic nonaperture group identifier.
+        identifier (str): Text string for a unique dynamic scene group identifier.
             This is required and cannot be None.
-        states(list[NonApertureState]): A list of nonaperture states.
+        states(list[NonApertureState]): A list of scene states.
 
     Properties:
-        * name
+        * identifier
         * states
+        * state_count
     """
 
-    def __init__(self, name, states):
-        self.name = name
+    def __init__(self, identifier, states):
+        self.identifier = identifier
         self.states = states
 
+    @property
+    def state_count(self):
+        """Number of states."""
+        return len(self.states)
+
     @classmethod
     def from_dict(cls, input_dict):
-        """Create a dynamic aperture from a dictionary.
+        """Create a aperture group from a dictionary.
```

Comment:
This is a `dynamic scene` not an `aperture group`. Also you should probably put some `Args:`